oggm/sandbox/run_alaska.py

```diff
@@ -154,34 +175,37 @@
 gdirs = workflow.init_glacier_regions(rgidf)
 
 if RUN_GIS_mask:
+    if PC:
+
+        Columbia_path = os.path.join(WORKING_DIR,
+                             'per_glacier/RGI50-01/RGI50-01.10/RGI50-01.10689/')
+        filename = os.path.join(Columbia_path, 'dem.tif')
+        dem_source = os.path.join(Columbia_path, 'dem_source.pkl')
+        grid_json = os.path.join(Columbia_path, 'glacier_grid.json')
+
+        # TODO : Remember to this paths on the cluster to run Columbia
+        Columbia_itmix = os.path.join(DATA_INPUT,
+                                      'RGI50-01.10689_itmixrun_new/')
+        dem_cp = os.path.join(Columbia_itmix, 'dem.tif')
+        dem_source_cp = os.path.join(Columbia_itmix, 'dem_source.pkl')
+        grid_json_cp = os.path.join(Columbia_itmix, 'glacier_grid.json')
+
+        # This is commented because we only need to replace the DEM once
+        # os.remove(filename)
+        # os.remove(dem_source)
+        # os.remove(grid_json)
+        # shutil.copy(dem_cp, filename)
+        # shutil.copy(dem_source_cp,dem_source)
+        # shutil.copy(grid_json_cp,grid_json)
 
-    Columbia_path = os.path.join(WORKING_DIR,
-                            'per_glacier/RGI50-01/RGI50-01.10/RGI50-01.10689/')
-    filename = os.path.join(Columbia_path,'dem.tif')
-    dem_source = os.path.join(Columbia_path,'dem_source.pkl')
-    grid_json = os.path.join(Columbia_path,'glacier_grid.json')
-
-    # TODO : Remember to this paths on the cluster to run Columbia
-    Columbia_itmix = os.path.join(DATA_INPUT,
-                                  'RGI50-01.10689_itmixrun_new/')
-    dem_cp = os.path.join(Columbia_itmix,'dem.tif')
-    dem_source_cp = os.path.join(Columbia_itmix,'dem_source.pkl')
-    grid_json_cp = os.path.join(Columbia_itmix,'glacier_grid.json')
-
-    os.remove(filename)
-    os.remove(dem_source)
-    os.remove(grid_json)
-    shutil.copy(dem_cp, filename)
-    shutil.copy(dem_source_cp,dem_source)
-    shutil.copy(grid_json_cp,grid_json)
-
-    #Calculate glacier masks
     execute_entity_task(tasks.glacier_masks, gdirs)
 
-# The rest Prepro tasks
+# Pre-processing tasks
 task_list = [
     tasks.compute_centerlines,
     tasks.initialize_flowlines,
+    tasks.compute_downstream_line,
+    tasks.compute_downstream_bedshape,
```

Comment:
Why do you need these? They are only necessary for doing a run

Reply:
mm I wasn't sure really what to take out or leave ... :P